### PR TITLE
Nick Smith React-Router-Movies

### PR DESCRIPTION
**Pull request by NickolausSmith**

### CODE REVIEW

File: client/src/Movies/Movie.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const id = this.props.match.params.id;
```
Comment (bdurb) [inline client/src/Movies/Movie.js]: 👍

File: client/src/Movies/MovieList.js [hunk 2]
```diff
@@ -34,6 +34,7 @@ export default class MovieList extends Component {
 function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
+<Link to={`/movies/${movie.id}`}>
```
Comment (bdurb) [inline client/src/Movies/MovieList.js]: 👍

File: client/src/App.js [hunk 3]
```diff
@@ -22,7 +22,11 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Link to="/">MovieList</Link>
```
Comment (bdurb) [inline client/src/App.js]: I'm curious why you decided to put a Route path as well as a link here.

**Comment by bdurb** [comment]
@NickolausSmith Thank you for the PR.  I also love movies,  but don't forget to be descriptive in your commits.

File: client/src/App.js (hunk 3 above)
Comment (NickolausSmith) [reply client/src/App.js]: Oh, must have left that in there by accident, from when I was toying with the links, thanks for the comments!

**Pull request closed without merging**